performance/buildSrc/src/main/kotlin/MPPTools.kt

```diff
@@ -60,6 +63,13 @@ fun defaultHostPreset(
         throw Exception("Host OS '$hostOs' is not supported in Kotlin/Native ${subproject.displayName}.")
 }
 
+fun getNativeProgramExtension(): String = when {
+    isMacos -> ".kexe"
+    isLinux -> ".kexe"
+    isWindows -> ".exe"
+    else -> TODO("Unknown host")
```

Comment:
`TODO` is for WIP code that you will implement later. `error` function is more appropriate here.